modules/nw-ingress-sharding-namespace-labels.adoc

```diff
@@ -11,6 +11,9 @@ Ingress Controller sharding by using namespace labels means that the Ingress
 Controller serves any route in any namespace that is selected by the namespace
 selector.
 
+.Ingress sharding using namespace labels
+image::nw-sharding-namespace-labels.png[Ingress sharding using namespace labels]
+
 Ingress Controller sharding is useful when balancing incoming traffic load among
 a set of Ingress Controllers and when isolating traffic to a specific Ingress
 Controller. For example, company A goes to one Ingress Controller and company B
```

Comment:
There's a `[WARNING]` on line 19 of this file, that feels like it doesn't belong, and isn't consistent with `modules/nw-ingress-sharding-route-labels.adoc`.